encoding/wkb/sql_example_test.go

```diff
@@ -74,3 +74,98 @@ func Example_value() {
 	// Output:
 	// 1 rows affected
 }
+
+func Example_scan_different_shapes() {
+	type Shape struct {
+		Name string
+		Geom wkb.Geom
+	}
+
+	db, mock, err := sqlmock.New()
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer db.Close()
+
+	mock.ExpectQuery(`SELECT name, ST_AsBinary\(geom\) FROM shapes`).
```

Comment:
I don't think `(` and `)` need to be escaped.

Reply:
If I remove them the test fail, I think it's related to an internal regexp usage in `go-sqlmock`. Moreover the other existing examples use the same escapes.

Reply:
OK, thanks.